docs/src/components/Footer/index.js

```diff
@@ -0,0 +1,176 @@
+import React, { useEffect, useState } from "react";
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+import Link from "@docusaurus/Link";
+
+const SocialMedia = () => {
+  const items = [
+    {
+      name: "facebook",
+      icon: require("@site/static/img/social_media/facebook.svg").default,
+      link: 'https://www.facebook.com/openebs',
+    },
+    {
+      name: "github",
+      icon: require("@site/static/img/social_media/github.svg").default,
+      link: 'https://github.com/openebs/openebs-docs',
+    },
+    {
+      name: "slack",
+      icon: require("@site/static/img/social_media/slack.svg").default,
+      link: 'https://openebs.io/community',
+    },
+    {
+      name: "linkedin",
+      icon: require("@site/static/img/social_media/linkedin.svg").default,
+      link: 'https://www.linkedin.com/company/openebs',
+    },
+    {
+      name: "youtube",
+      icon: require("@site/static/img/social_media/youtube.svg").default,
+      link: 'https://www.youtube.com/channel/UC3ywadaAUQ1FI4YsHZ8wa0g',
+    },
+    {
+      name: "twitter",
+      icon: require("@site/static/img/social_media/twitter.svg").default,
+      link: 'https://twitter.com/openebs',
+    },
+  ];
+  return (
+    <div className="row">
+      {items?.map((item) => {
+        return (
+          <Link className="link_icon" key={item?.name} to={item?.link}>
+            <item.icon />
+          </Link>
+        );
+      })}
+    </div>
+  );
+};
+
+export const Footer = () => {
+  const { siteConfig } = useDocusaurusContext();
+  const openebsLogo = siteConfig?.customFields?.openebsLogo;
+  const gettingStarted = siteConfig?.customFields?.footbarLinks?.gettingStarted;
+  const contactUs = siteConfig?.customFields?.footbarLinks?.contactUs;
+  const privacyPolicy = siteConfig?.customFields?.footbarLinks?.privacyPolicy;
+  const githubApiContributors = siteConfig?.customFields?.githubApiContributors;
+  const copyRight = siteConfig?.themeConfig?.footer?.copyright;
+  const [isLoaded, setIsLoaded] = useState(false);
+  const [items, setItems] = useState([]);
+  //getting the top contributors from github by sending the api order as desc
+  useEffect(() => {
+    fetch(
+        githubApiContributors
+    )
+      .then((res) => res?.json())
+      .then(
+        (result) => {
+          setIsLoaded(true);
+          setItems(result);
+        },
+        (error) => {
+          setIsLoaded(true);
+          console.error(error);
+        }
+      );
+    return () => {
+      setItems([]);
+    };
+  }, []);
```

Comment:
empty brackets can be removed.

Reply:
Using `[]`,  useEffect will run only one-time